api/settings/base.py

```diff
@@ -145,6 +146,7 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/dev/howto/static-files/
 
+STATIC_ROOT = "/opt/docker_polls_group/api/static/"
```

Comment:
please use `BASE_DIR, static`
BASE_DIR = /opt/docker_polls_group/api/
BASE_DIR is folder where we have manage.py.

Reply:
@romaoksenyuk 
STATIC_URL = url for found some static files
STATIC_ROOT = all files will be copied when you make on server `collectstatic` command (in Prod)
STATICFILES_DIRS = if you run `collectstatic` command django found files from paths and copy to STATIC_ROOT folder (in Prod)
[DOCS](https://docs.djangoproject.com/en/2.2/howto/static-files/).